gamemodes/terrortown/gamemode/client/cl_voice.lua

```diff
@@ -51,6 +56,18 @@ end
 
 CreateVoiceTable()
 
+local function ComboBoxChoicesFromKeys(tbl, labelPrefix, default)
+    local choices = {}
+    for key in pairs(tbl) do
+        choices[#choices + 1] = {
+            title = LANG.TryTranslation(labelPrefix .. key),
+            value = key,
+            select = key == default,
+        }
+    end
+    return choices
+end
+
```

Comment:
This function is not voice-specific and could be a general client library function.
I didn't know where to put those kind of functions though so I left it as a local function.

Reply:
maybe put it in `util`?